package.json

```diff
@@ -24,8 +24,8 @@
   },
   "homepage": "https://github.com/WordPress/wporg-two-factor#readme",
   "dependencies": {
-    "@wordpress/base-styles": "^4.17.0",
-    "@wordpress/env": "^10.1.0"
+    "@wordpress/base-styles": "^5.2.0",
+    "@wordpress/env": "^10.2.0"
```

Comment:
And the tests pass for e with just the babel and jest configs, these updates don't seem to be required, but maybe there's value in them.

Reply:
Reverted them, since I didn't test.